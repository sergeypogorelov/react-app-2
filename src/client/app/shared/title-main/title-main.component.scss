@import '../../../styles/variables';

.title-main {
  &-prefix {
    font-weight: bold;
  }
  font-family: $f-medium;
  font-size: 20px;
  color: $c-primary;
}
