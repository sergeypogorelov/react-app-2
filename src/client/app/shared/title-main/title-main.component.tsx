import './title-main.component.scss';

import React, { FunctionComponent } from 'react';

export const TitleMain: FunctionComponent<{}> = () => {
  return (
    <p className="title-main m-0">
      <span className="title-main-prefix">netflix</span>
      <span>roulette</span>
    </p>
  );
};
