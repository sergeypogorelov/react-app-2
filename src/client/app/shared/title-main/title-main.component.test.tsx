import React from 'react';
import renderer from 'react-test-renderer';

import { TitleMain } from './title-main.component';

test('TitleMain component should render correctly', () => {
  const tree = renderer.create(<TitleMain />).toJSON();
  expect(tree).toMatchSnapshot();
});
