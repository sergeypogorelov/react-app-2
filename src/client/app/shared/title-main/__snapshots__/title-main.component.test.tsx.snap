// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleMain component should render correctly 1`] = `
<p
  className="title-main m-0"
>
  <span
    className="title-main-prefix"
  >
    netflix
  </span>
  <span>
    roulette
  </span>
</p>
`;
